CHANGELOG.md

```diff
@@ -24,11 +24,37 @@ This changelog includes all versions and major variants of the mod going all the
 > * **TM:PE v11 LABS** - TM:PE versions 11.0 to 11.5.0 (test releases)
 > * **[TM:PE v11 STABLE](https://steamcommunity.com/sharedfiles/filedetails/?id=1637663252) - TM:PE versions 11.0 and above (stable releases)**
 > * **[TM:PE v11 TEST](https://steamcommunity.com/sharedfiles/filedetails/?id=2489276785) - TM:PE versions 11.6.0 and above (test releases)**
->  
-> Date format: dd/mm/yyyy
 
 </details>
 
+> Date format: dd/mm/yyyy
+
+#### TM:PE V11.6.5.2 TEST, 18/04/2022
+
+- [Meta] This release improves lane routing, and fixes some bugs.
```

Comment:
also lane connector UI/back-end was improved a little bit. Does not count as lane routing from user point of view?

Reply:
yup, I tweaked the text to include note on UI and the other stuff is listed in more detail elsewhere in the list